UPGRADE-3.x.md

```diff
@@ -1,6 +1,11 @@
 UPGRADE 3.x
 ===========
 
+UPGRADE FROM 3.1.2 to 3.x
```

Comment:
Please do not add upgrade title. Just add the note on the `3.x` section.

Reply:
@Soullivaneuh done.